src/ad-engine/utils/iframe-builder.ts

```diff
@@ -1,10 +1,16 @@
-import { AdSlot } from '../models';
-
 export class IframeBuilder {
-	create(adSlot: AdSlot): HTMLIFrameElement {
-		const doc = adSlot.getElement();
+	create(element: HTMLElement, htmlContent: string = null): HTMLIFrameElement {
```

Comment:
Why default to null?
```suggestion
	create(element: HTMLElement, htmlContent?: string): HTMLIFrameElement {
```

Reply:
👍